src/app/components/util/internal-scroll-button.jsx

```diff
@@ -31,12 +38,16 @@ const InternalScrollButton = (props) => {
     target.scrollIntoView({ behavior: 'smooth', block: 'start' });
   };
 
+  const classes = useStyles();
+
   return (
     <Button
       onClick={handleClick}
-      aria-label={`Scroll display to ${name}`}
+      aria-label={`Scroll display to ${children}`}
```

Comment:
This probably won't work like this, as the `{children}` thing needs to be a string, and in these cases it's a string plus an Icon. I think we either need a text value that's passed in as a prop, or maybe we don't need the attribute at all, though I'm not positive about that

Reply:
I'm not sure how you want to go about this, but lets make a plan for how we apply an icon to buttons types because im sure this will pop up again. If you could walk me thru how you want to apply this I can start using the method moving forward.